src/admin/containers/CreateEventForm.tsx

```diff
@@ -64,4 +63,4 @@ const FormikCreateEventForm = withFormik({
   },
 })(CreateEventForm as React.FC);
 
-export default connect(null, { postEvent, copyLink })(FormikCreateEventForm);
+export default connect(null, { postEvent })(FormikCreateEventForm);
```

Comment:
The only instances where I left `connect` was with Formik since there isn't a way to access the `useDispatch` or `useSelector` hooks with the `withFormik` HOC. In order to remove the `connect` usage there, we'd have to rewrite the Formik code with Formik hooks or different component APIs which is probably better suited for another PR.